package.json

```diff
@@ -98,10 +88,8 @@
     "release-it": "^18.1.2",
     "source-map-support": "^0.5.21",
     "ts-node": "^10.9.2",
-    "ts-node-register": "^1.0.0",
     "tsx": "^4.19.2",
     "typedoc": "^0.25.1",
-    "typedoc-plugin-markdown": "^3.15.2",
```

Comment:
Let's keep this, since it is used in the docs generation script https://github.com/oceanprotocol/ocean.js/blob/d2d54a5d5077c986a2051016ea94bfd96e81660c/package.json#L27